module-2/lab-matplotlib-seaborn/your-code/challenge-2.ipynb

```diff
@@ -23,18 +23,17 @@
   },
```

Comment:
- You can run the following to plot a barplot of median steps by weekend and workdays. 
```Python
# Median Steps by Work_or_Weekend
plt.sca(axes[1])
weekday_steps = fitbit['Steps'].groupby(fitbit['Work_or_Weekend']).median().sort_values()
weekday_steps.plot(kind = 'bar',alpha = 0.5)
plt.ylabel('Median number of steps')
plt.title('Median Steps walked by Workdays and Weekend')
plt.xticks(list(range(2)),['Weekend','Workdays'])
plt.savefig('steps_work_weekend.png',bbox_inches='tight')
```
- You can set the following to get your graphs horizontally. Sometimes it's useful. 
```Python 
nrows = 1, ncols = 3
````